E2a-TRST/TSTP/README.md

```diff
@@ -0,0 +1,179 @@
+  - *IfcAlignmentVertical*
+  - *IfcAlignmentSegment*
+
+- For this test instruction:
+  - *IfcRailwayPart*
+  - *IfcElementAssembly*
+  - *IfcTrackElement*
+  - *IfcRail*
+  - *IfcMechnicalFastner*
+  - *IfcReferent*
+
+
+</details>
+
+<details><summary>Concept Templates</summary>
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+- Inherited from imported tests:
+  - *Project Units*
+  - *Project Representation Context*
+  - *Project Global Positioning*
+  - *Spatial Decomposition*
+  - *Spatial Composition*
+  - *Alignment Layout*
+  - *Alignment Geometry Cant*
+  - *Spatial Container*
+  - *Product Local Placement*
+  - *Object User Identity*
+  - *Revision Control*
+  - *Software Identity*
+  - *Axis Geometry*
+  
+- For this test instruction:
+  - *Product Linear Placement*
+  - *Object Predefined Type*
+  - *Element Composition*
+  - *Element Decomposition*
+  - *Product Relative Positioning*
+  - *Product Span Positioning*
+  - *Spatial Containment*
+  - *Body AdvancedSwept Directrix Geometry*
+  - *Body Tessallated Geometry*
+  - *Object Typing*
+  - *Type Body Tessellated Geometry*
+
+</details>
+
+## Test Case Imports
+Test instructions are defined with a modular approach to reduce repetition of validation criteria and test content, and improve vendors ability to solve issues and bugs. therefore this test instruction *imports/reuses* the following Test instructions and entities with the relevant associated validation criteria.
+
+:construction: under construction :construction:
+
+<details><summary>Imports & Reuses</summary>
+
+| TI Code                                  | Test Instruction Title    | Comments                     |
+|------------------------------------------|---------------------------|------------------------------|
+| [IFC4.3AbRV_E1b_ALCT2](../../E1b-ARCT/ALCT2) | Alignment Railway Cant 2  | As outlined in the dataset Imported Entities Table |
+
+</details>
+
+## Usages, Constraints & Logic
+The following itemised restrictions and constraints shall be placed on IFC Entities & Concept Templates:
+
+:construction: under construction :construction:
+
+<details><summary>Semantic Usages, Constraints & Logic</summary>
+
+The following itemised Usages, Constraints & Logic are normative entries within the AbRV MVD and MUST be satisfied to meet the defined validation criteria
+
+- IfcElementAssembly
+    - *There exists 1 IfcElementAssembly with PredefinedType set to TRACKTURNOUTPANEL*
+	- *Each IfcElementAssembly/TRACKTURNOUTPANEL shall have a linear placement relative to the alignment*
+	- *Each IfcElementAssembly/TRACKTURNOUTPANEL is spatially contained in an IfcRailwayPart/TRACKSUPERSTRUCTURE*
+	- *Each IfcElementAssembly/TRACKTURNOUTPANEL shall have a product span placement*
```

Comment:
What is *product span placement*? Can you point me to the documentation?

Reply:
Here: http://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Object_Connectivity/Product_Span_Positioning/content.html

Reply:
> I may have missed somewhere in the readme: Regarding E2a-TRST/TSTP/Dataset/Sleeper: is it expected to have the 3D model according to the real situation or the simplified situation?

Stated in Dataset/README.md line 30:
- The geometry of IfcTrackElement.SLEEPER can be simplified and placed according to the drawing in Fig. 6.